src/expr/scalar/func.rs

```diff
@@ -625,6 +625,17 @@ fn round_decimal<'a>(a: Datum<'a>, b: Datum<'a>, a_scale: u8) -> Datum<'a> {
     Datum::from(decimal.round(round_to).significand())
 }
 
+fn convert_from<'a>(a: Datum<'a>, b: Datum<'a>) -> Datum<'a> {
+    if b.unwrap_str().to_lowercase() != "utf8" {
+        panic!("Materialize only supports convert_from() with utf8");
+    }
+
+    match from_utf8(a.unwrap_bytes()) {
+        Ok(from) => Datum::String(from),
```

Comment:
We mustn’t panic here or above! Instead add new EvalError variants, and return those.